asset-transfer-basic/chaincode-java/README.md

```diff
@@ -0,0 +1,160 @@
+
+## Basic asset transfer 
+
+This project demonstrates the use of the Java SDK, running a basic asset transfer contract using the "chaincode as a service" 
+pattern.
+
+
+## Setup 
+
+In this sample we will employ the [Kubernetes Test Network](../../test-network-k8s) to illustrate a scenario of 
+building, running, and debugging chaincode on a development workstation.  
+
+This project is also compatible with the legacy chaincode builder pipeline and the compose based test-network.
+For additional details, see the [End-to-end with the test-network](../../test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md#end-to-end-with-the-the-test-network)
+documentation.
```

Comment:
It's great to have a link to the relevant test-network however I think the test-network-k8s instructions below would also be better in its own (k8s) specific doc rather than inline in the chaincode readme, e.g. a `JAVA_CHAINCODE_DEBUG.md` file in [test-network-k8s/docs](https://github.com/hyperledger/fabric-samples/tree/main/test-network-k8s/docs). Also "the legacy chaincode builder pipeline and the compose based test-network" doesn't sound right- the builder pipeline is the same in both test networks isn't it?